### ISSUE TITLE
Tables without borders

### ISSUE THREAD

**Issue by SeniorSpielbergo** [WordPress/gutenberg#16840]
**Is your feature request related to a problem? Please describe.**
When using the table block in Gutenberg I currently can _either_ choose between a style using borders with a white background _or_ a style with stripes (alternating white and grey background) without borders. I am currently missing an option to use a table _without_ borders and with a plain white background.

**Describe the solution you'd like**
It would be great to have a style for the table block without borders or an option to disable borders for any selected table style.

**Comment by afercia** [comment]
@SeniorSpielbergo thanks for your report. Would you mind sharing more details on the specific use case for your proposer enhancement?

Worth reminding the Table block should be used for data tables. Gutenberg discourages the usage of layout tables. For layout, there are better options e.g. the Columns block.

<img width="355" alt="Screenshot 2019-08-01 at 09 28 27" src="https://user-images.githubusercontent.com/1682452/62273885-f1403600-b43e-11e9-9e17-4ded7f96a131.png">

**Comment by SeniorSpielbergo** [comment]
To be honest, my current use case would probably fall into the categorie of layout tables. However, I can imagine that adding the option to disable the borders would give additional flexibility. Thus, I think this could at least be discussed as part of the currently undergoing improvements to the table block (cf. https://github.com/WordPress/gutenberg/projects/33).

**Comment by talldan** [comment]
Closing this in favour of #27651
